webapp/src/main/webapp/news/components/NewsApp.vue

```diff
@@ -67,9 +67,9 @@
                   <span class="viewsText">{{ news.viewsCount }}  {{ $t('news.app.views') }}</span>
                 </p>
               </div>
-              <div class="share pull-right">
-                <exo-news-share-activity :activity-id="news.activityId" :news-id="news.newsId" :news-title="news.newsTitle">
-                </exo-news-share-activity>
+              <div class="newsRightActions pull-right">
```

Comment:
```suggestion
              <div class="newsRightActions">
```
Since the dispplay has been switched to flex, the pull-right has no effect now IMO.

Reply:
done